modules/apps/frontend-icons/frontend-icons-web/src/main/resources/META-INF/resources/js/index.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+const ICONS_CONFIG = (window as any)._FRONTEND_ICONS;
```

Comment:
:thinking: If nobody is going to access `window._FRONTEND_ICONS` because you are abstracting it with the API exported from this module, you can store it here (inside the module) maybe. That would make it truly private and would make collisions impossible, for example.

The "downside" is that if you put it inside a module, it won't be accessible from "legacy" JavaScript. But given that this is a new API, we may try...

Reply:
So my primary reason for putting it on the window was because I wanted to only set the constants in one place, the java util file. I didn't want to duplicate the same hard coded string references in both js and java files, so I thought by only storing it in the java and then setting on the window, we would have a single source of truth.

One alternative would be to nix that whole idea and hardcode the path `/o/icons` and system pack name `clay` in both the js and java files but then add comments to reference they are hardcoded in two places.